src/api/serviceJourney.ts

```diff
@@ -7,11 +6,10 @@ import {ServiceJourneyWithEstCallsFragment} from '@atb/api/types/generated/fragm
 
 export async function getServiceJourneyWithEstimatedCalls(
   id: string,
-  date: Date,
+  isoDate: string,
```

Comment:
Could consider `string | Date` as type, and do the `formatToIsoDate` here. There is a principle that sounds something like "Don't make the client do what the API can do for them", although I can't remember the exact phrasing, but I think that might be a good fit here. At least I think it is architecturally ok for the api-module to use date utils.

Reply:
This made me look into this a bit closer, and I think the correct thing to do here is to remove date conversion all together, and rely on the serviceDate string. We don't need the date object at any point, and might even lead to time zone bugs.

Updated in 692676580fb58cd69e62293abd28ce94ea9e9443

Reply:
This fails in `date.test.ts`, so I think we might have had a bug here!

```ts
  describe('date conversion', () => {
    let datestring = '2024-01-25T23:00:00Z';
    let date = new Date(datestring);
    expect(formatISO(date, {representation: 'date'})).toBe('2024-01-25');
  });
```